src/services/routes/routes.svc.coffee

```diff
@@ -9,7 +8,7 @@ angular
     #=======================================
     defaults =
       # base paths
-      mnoHub: '/api/v2'
+      mnoHub: 'http://localhost:7000/mnoe/jpi/v1'
       impacPrefix: '/impac'
```

Comment:
Do we still have a case where `impacPrefix` wouldn't be `/impac` ? I think this was done for Maestrano and `/analytics` ?...
If the case does no longer exist I suggest to simply remove the variable or to add it to the `mnoHub` path variable